### ISSUE TITLE
New binaries only work in local mode

### ISSUE THREAD

**Issue by CraigEMP** [rickparrish/Usurper#16]
The new Windows binaries created in issue #6 only work in local mode, regardless of the arguments passed on the command line.

**Comment by rickparrish** [comment]
At one point I switched from the original DDPlus routines to my own doorkit RMDoor, which takes different command-line parameters.  I have a feeling it was only done in the GitHub binaries, not in whatever the last release was, so the parameters between the last release and the GitHub binaries will be different.

If I remember right you just need to pass /D with the dropfile, ie USURPER /DC:\BBS\NODE1\DOOR32.SYS

If that doesn't work let me know and I'll take a closer look.  Haven't had a BBS setup to test it with in awhile, but it might be a good excuse to try the latest Synchronet!

**Comment by CraigEMP** [comment]
Ah, ok.  Looks like /D is undocumented, unless I completely missed it somewhere.  Anyway, I can confirm it works, which is great.  Thanks!

In Synchronet config, I set "Native Executable" to yes, and "Use Shell to Execute" also to yes (if shell is set to no, interestingly, it launches the game but doesn't send the output to the user session in progress).

While running, I've also noticed all of the items listed by @stack-fault in Issue #14, so it appears it's not unique to the Linux build.
